azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/action/NetworkAction.kt

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.chat.redux.action
+
+import org.threeten.bp.OffsetDateTime
+
+internal sealed class NetworkAction : Action {
+    class Connected : NetworkAction()
+    class Disconnected : NetworkAction()
+    class SetDisconnectedOffset(val disconnectOffsetDateTime: OffsetDateTime) : NetworkAction()
```

Comment:
Can Time be Implicit or pass the disconnected time in the above action?

Reply:
discussed, no action required